source/mainnet/net/voting/coordinator.rst

```diff
@@ -0,0 +1,123 @@
+=========================
+
+The coordinator tool is an administrative tool to coordinate the parts of the election that are not done by voters or guardians.
+
+The coordinator of the election is not trusted per se. All the actions it can do can be verified by any other party. The coordinator tool can be mostly run without access to any secrets to verify results of the election. However some commands take "admin keys" where the result of operations can be posted in the contract for everybody to see. If such a result is present in the contract and another user runs the coordinator tool, the tool will check that the result it computed and the one posted in the contract agree.
+
+Build and run
+=============
+
+To build the tool make sure you have the repository submodules initialized:
+
+.. code-block:: console
+
+    git submodule update --init --recursive
+
+Build the tool by running:
+
+.. code-block:: console
+
+    cargo build --release
+
+This will produce a single binary election-coordinator in target/release directory.
+
+Commands
+========
+
+The tool has the following subcommands. All commands have a ``--help`` option which explains the input and output parameters.
+
+Get the list of initial weights
+-------------------------------
+
+Use the ``initial-weights`` command to gather the average amount of CCD on an account in the given period. The intention is that this command is used to produce the initial weights of each account prior to the election starting. The output of this command is a CSV file used in the ``final-weights`` command after the election.
+
+Example
+
+.. code-block:: console
+
+    election-coordinator --node http://localhost:20001 initial-weights  --start 2024-01-01T00:00:00Z --end 2024-01-03T00:00:00Z --out initial-weights.csv
+
+The weights are stored in the initial-weights.csv file.
+
+Create a new election instance
+------------------------------
+
+Use the ``new-election`` command to create the necessary files and the contract for a new election. In particular it will:
+
+- create an election manifest based on the inputs
+
+- create election parameters based on the inputs
+
+- create a new smart contract instance.
+
+.. code-block:: console
+
+    election-coordinator new-election --module ../contracts/concordium-governance-committee-election/concordium-out/module.wasm.v1 --threshold 1 --admin ../test-scripts/keys/2yJxX711aDXtit7zMu7PHqUMbtwQ8zm7emaikg24uyZtvLTysj.export --election-start '2024-02-01T00:00:00Z' --election-end '2024-02-07T00:00:00Z' --decryption-deadline '2024-02-08T00:00:00Z' --delegation-string 'delegatevote2024' --out election --voters-file initial-weights.csv --guardian 31bTNa42u1zZWag2bknEy7VraeJUozXsJMN1DFjQp7E5YR6a3G --guardian 4PF6BH8bKvM48b8KNYdvGW6Sv3B2nqVRiMnWTj9cvaNHJQeX3D --candidate 'http://candidates/candidate1.json' --candidate 'http://candidates/candidate2.json' --node 'https://grpc.testnet.concordium.com:20000' --base-url https://gcvoting.testnet.concordium.com`
```

Comment:
Might have come from my previous suggestion, but the candidate URL's look a little weird. This would be better to reflect the duality of how they can be specified (i.e. path or URL)

```suggestion
    election-coordinator new-election --module ../contracts/concordium-governance-committee-election/concordium-out/module.wasm.v1 --threshold 1 --admin ../test-scripts/keys/2yJxX711aDXtit7zMu7PHqUMbtwQ8zm7emaikg24uyZtvLTysj.export --election-start '2024-02-01T00:00:00Z' --election-end '2024-02-07T00:00:00Z' --decryption-deadline '2024-02-08T00:00:00Z' --delegation-string 'delegatevote2024' --out election --voters-file initial-weights.csv --guardian 31bTNa42u1zZWag2bknEy7VraeJUozXsJMN1DFjQp7E5YR6a3G --guardian 4PF6BH8bKvM48b8KNYdvGW6Sv3B2nqVRiMnWTj9cvaNHJQeX3D --candidate candidate1.json --candidate https://some.url/candidate2.json --node 'https://grpc.testnet.concordium.com:20000' --base-url https://gcvoting.testnet.concordium.com`
```